plugins/RocketsPlugin/RocketsPlugin.cpp

```diff
@@ -558,7 +558,6 @@ class RocketsPlugin::Impl
 
         _handleFrameBuffer();
         _handleSimulationHistogram();
-        _handleVolumeHistogram();
```

Comment:
shouldn't we remove the entire code then? I'm still not sure this is the right thing to do. As long as the client doesn't request the histogram, this shouldn't matter. I will push a change to the VizTools where the states are requested on first access, so this might fix the problem you experienced.

Reply:
In a later PR, with volume refactoring.